src/dash/parser/matchers/StringMatcher.js

```diff
@@ -32,40 +32,41 @@
  * @classdesc Matches and converts xs:string to string, but only for specific attributes on specific nodes
  */
 import BaseMatcher from './BaseMatcher';
+import DashConstants from '../../constants/DashConstants';
 
 class StringMatcher extends BaseMatcher {
     constructor() {
         super(
             (attr, nodeName) => {
                 const stringAttrsInElements = {
-                    'MPD':                        [ 'id', 'profiles' ],
-                    'Period':                     [ 'id' ],
```

Comment:
Meant to comment on the addition rather than the removal but you get the idea ...